src/main/java/SystemDisplay.java

```diff
@@ -0,0 +1,56 @@
+public class SystemDisplay implements Display {
+
+    public void write(String message) {
+        System.out.println(message);
+    }
+
+    public String read() {
+        return null;
```

Comment:
This method returning null looks like a sign that perhaps the method read does not belong on the Display interface. Maybe it should be separated out?